cmd/e2e/src/tests/pageObjects/loginPage.ts

```diff
@@ -0,0 +1,41 @@
+// Copyright 2024 Specter Ops, Inc.
+//
+// Licensed under the Apache License, Version 2.0
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+import { type Page } from '@playwright/test';
+
+export default class LoginPage {
```

Comment:
I like this pattern! Is the idea that when we want to interact with a given page, we add a to `pageObject/` and can reuse those page interactions across different tests? 

A small organizational suggestion: we move `pageObjects/` under `helpers/` because these arent exactly tests but instead utils for tests.